diff --git a/config.h b/config.h
index baf9420..7afaa3e 100644
--- a/config.h
+++ b/config.h
@@ -8,10 +8,10 @@ static const Block blocks[] = {
 	{"dwm_kernel", 0, 0},
 	{"dwm_cpu_temp", 3, 0},
 	// { "dwm_date", 3600, 0 },
-	{"dwm_brightness", 0, 11},
+	// {"dwm_brightness", 0, 11},
 	{"dwm_volume", 0, 10},
 	{"dwm_network", 3, 0},
-	{"dwm_battery", 3, 0},
+	// {"dwm_battery", 3, 0},
 	{"dwm_clock", 3, 0},
 	{"dwm_keyboard_layout", 0, 12},
- 	{"dwm_uptime", 60, 0},
+	{"dwm_uptime", 10, 0},
