diff --git a/config.h b/config.h
index baf9420..1a68cac 100644
--- a/config.h
+++ b/config.h
@@ -2,17 +2,17 @@
 
 static const Block blocks[] = {
 	/* Command *//* Update Interval *//* Update Signal */
-	{"dwm_traffic", 1, 0 },
-	{"dwm_memory", 3, 0},
+	// {"dwm_traffic", 1, 0 },
+	{"dwm_memory", 4, 0},
 	{"dwm_disk", 5, 0},
 	{"dwm_kernel", 0, 0},
-	{"dwm_cpu_temp", 3, 0},
+	{"dwm_cpu_temp", 4, 0},
 	// { "dwm_date", 3600, 0 },
 	{"dwm_brightness", 0, 11},
 	{"dwm_volume", 0, 10},
 	{"dwm_network", 3, 0},
-	{"dwm_battery", 3, 0},
-	{"dwm_clock", 3, 0},
+	{"dwm_battery", 10, 0},
+	{"dwm_clock", 5, 0},
 	{"dwm_keyboard_layout", 0, 12},
+	// {"dwm_uptime", 60, 0},
 };
